frontend/src/pages/home.tsx

```diff
@@ -70,7 +70,12 @@ const Home: FC = () => {
   }
 
   return (
-    <Layout breadcrumbItems={[]} contained={false}>
+    <Layout
+    breadcrumbItems={[]}
+    contained={false}
+    hideFooter='never'
+    hideHeader='never'
```

Comment:
Remove it if it's `never` (undefined equals 'never') and assign Layout's defaultProps:
```tsx
Layout.defaultProps: { 
  hideFooter: 'never'
  hideHeader: 'never'
}

Reply:
Updated